Python/pywarpx/picmi.py

```diff
@@ -1489,6 +1489,11 @@ class Simulation(picmistandard.PICMI_Simulation):
     warpx_zmax_plasma_to_compute_max_step: float, optional
         Sets the simulation run time based on the maximum z value
 
+    warpx_compute_max_step_from_btd: integer, optional
```

Comment:
Is this technically a `bool` rather than an `integer`?
```suggestion
    warpx_compute_max_step_from_btd: bool, default=0
```